### DISCUSSION TITLE
post/naudio-midi-playback-soundfont-meltysynth

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#68] [Announcements]
# post/naudio-midi-playback-soundfont-meltysynth

Mark Heath's Development Blog

https://markheath.net/post/naudio-midi-playback-soundfont-meltysynth

**Comment by jonjonsson** [comment]
Cool! Do you think there will be a recording of your talk?

**Reply by markheath** [reply]
yes, it's an NDC event, so usually they post the videos on their channel a couple of months later

**Reply by HarryLau2320** [reply]
Hello Mr. Mark, is it possible to run NAudio lib from Android Platform?
